code/modules/jobs/job_types/security.dm

```diff
@@ -124,7 +124,7 @@ Detective
 
 	outfit = /datum/outfit/job/detective
 
-	access = list(access_sec_doors, access_forensics_lockers, access_morgue, access_maint_tunnels, access_court, access_brig, access_weapons)
+	access = list(access_sec_doors, access_security, access_forensics_lockers, access_morgue, access_maint_tunnels, access_court, access_brig, access_weapons)
```

Comment:
there is plenty of incentive to play security, if we make it clear that detective still has to take orders from sec and has no authority over them, additionally, in roles like medbay if you choose medical doctor on skeleton crew you still get chem access, you aren't told the medivends and the store room are enough, because on low pop, a stick and a non lethal six shooter is not gonna stop the tag team of rampaging changelings